pkg/storaged/details.js

```diff
@@ -63,7 +63,7 @@
             mdraid_add_disk: function mdraid_add_disk(path) {
                 var mdraid = client.mdraids[path];
 
-                dialog.open({ Title: _("Add Disks"),
+                dialog.open({ Title: _("Add Spare Disks"),
```

Comment:
This title can be misleading. If a RAID is degraded (missing disks), the disk I want to add might not be a spare at all, but essential.